data_steward/deid/config/ids/tables/measurement.json

```diff
@@ -1,8 +1,28 @@
         }
```

Comment:
not finding the alias `measurement` in `measurement.measurement_id = record2.measurement_id`